lessons/02-javascript/README.md

```diff
@@ -331,10 +338,10 @@ $ node functions.js
 This is totally legal. It might help to break down the expression we just printed:
 
 ```node
-func1()(); // first, func1 is called
+func1()(); // first, func1 is called and we say we will call its output
```

Comment:
(Typically, most jQuery examples all start with an IIFE, so it'd be good to understand what's going on there)